packages/common/src/types/search.ts

```diff
@@ -1,11 +1,22 @@
+/**
+ * Search operator
+ * @enum {string}
+ * @property {string} Eq - Equal
+ * @property {string} Ne - Not equal
+ * @property {string} Gt - Greater than
+ * @property {string} Lt - Less than
+ * @property {string} Gte - Greater than or equal
+ * @property {string} Lte - Less than or equal
+ * @property {string} Contain - Contain
+ */
 export enum ESearchOperator {
-  Eq,
-  Ne,
-  Gt,
-  Lt,
-  Gte,
-  Lte,
-  Contain,
+  Eq = 'Eq',
+  Ne = 'Ne',
+  Gt = 'Gt',
+  Lt = 'Lt',
+  Gte = 'Gte',
+  Lte = 'Lte',
+  Contain = 'Contain',
```

Comment:
I think it should be lower case since we pass it into mongo query
![Uploading image.png…]()